Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteLayerTreeDrawingAreaProxyMac.mm

```diff
@@ -304,13 +304,19 @@ explicit RemoteLayerTreeDisplayLinkClient(WebPageProxyIdentifier pageID)
     RetainPtr<CABasicAnimation> renderViewAnimationCA = DrawingArea::transientZoomSnapAnimationForKeyPath("transform"_s);
     NSValue *transformValue = [NSValue valueWithCATransform3D:transform];
     [renderViewAnimationCA setToValue:transformValue];
+    RefPtr<WebPageProxy> protectedWebPageProxy = &m_webPageProxy;
     
-    [CATransaction setCompletionBlock:[layerForPageScale, this, scale, constrainedOrigin, transform] () {
-        layerForPageScale.transform = transform;
-        [layerForPageScale removeAnimationForKey:transientAnimationKey];
-        [layerForPageScale removeAnimationForKey:@"transientZoomCommit"];
-        m_transactionIDAfterEndingTransientZoom = nextLayerTreeTransactionID();
-        m_webPageProxy.send(Messages::DrawingArea::CommitTransientZoom(scale, constrainedOrigin), m_identifier);
+    [CATransaction setCompletionBlock:[scale, constrainedOrigin, transform, protectedWebPageProxy] () {
```

Comment:
You can write this as: `[..., protectedWebPageProxy = Ref { m_webPageProxy }]`

Reply:
Fixed.